src/views/Country/index.tsx

```diff
@@ -42,18 +42,28 @@ export function Component() {
         pending: countryResponsePending,
         response: countryResponse,
     } = useRequest({
-        skip: !countryId,
+        skip: isNotDefined(countryId),
         url: '/api/v2/country/{id}/',
         pathVariables: {
             id: Number(countryId),
         },
     });
 
+    const {
+        pending: countrySnippetPending,
+        response: countrySnippetResponse,
+    } = useRequest({
+        skip: isNotDefined(countryId),
+        url: '/api/v2/country_snippet/',
```

Comment:
This response is not used in the current view, lets move it where it is needed (i.e. to `CountryAdditionalData`